### PR TITLE
add django-localflavor to conda-forge

### PR DESCRIPTION
**Pull request by kcemenike**
Checklist
- [ x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ x] Source is from official source
- [ x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ x] Build number is 0
- [ x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/django-localflavor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 24e9a03bc11ced92365821d3cfdd96eb4e2f3e44a6f40e86c645d3e0ec65ec53
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (dopplershift) [inline recipes/django-localflavor/meta.yaml:18]: I don't think this package contains any C code so I think the `compiler` (and the whole `build` section) can go.

File: recipes/django-localflavor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  description: |
+  Most of the localflavor add-ons are localized form components deriving from the forms framework – for example, a USStateField that knows how to validate U.S. state abbreviations, and a FISocialSecurityNumber that knows how to validate Finnish social security numbers.
+  
+  doc_url: https://django-localflavor.readthedocs.io/
+  dev_url: https://github.com/django/django-localflavor/
+
+extra:
+  recipe-maintainers:
+    - django
```
Comment (dopplershift) [inline recipes/django-localflavor/meta.yaml:45]: `django` is the name of the project and not a user who can have merge rights on this repository, so should be removed.

File: recipes/django-localflavor/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (dopplershift) [inline recipes/django-localflavor/meta.yaml:24]: I think you're missing some things here that are in the project's `setup.py`:
```py
        'django>=2.2',
        'python-stdnum>=1.6',
```

File: recipes/django-localflavor/meta.yaml (hunk 2 above)
Comment (kcemenike) [reply recipes/django-localflavor/meta.yaml]: done

File: recipes/django-localflavor/meta.yaml (hunk 1 above)
Comment (kcemenike) [reply recipes/django-localflavor/meta.yaml]: build section removed, thanks!

File: recipes/django-localflavor/meta.yaml (hunk 3 above)
Comment (kcemenike) [reply recipes/django-localflavor/meta.yaml:22]: apologies, that has been resolved

**Comment by dopplershift** [comment]
Looks like `python-stdnum` has not yet been packaged for conda-forge. You'll need to do that (you can include it in this PR).

**Comment by kcemenike** [comment]
> Looks like `python-stdnum` has not yet been packaged for conda-forge. You'll need to do that (you can include it in this PR).

Thanks a lot for this @dopplershift I didn't realise till the test returned an error.

**Comment by kcemenike** [comment]
> Looks like `python-stdnum` has not yet been packaged for conda-forge. You'll need to do that (you can include it in this PR).

@dopplershift python-stdnum has now been packaged with this PR

**Comment by dopplershift** [comment]
Thanks and congrats on your first set of recipes!

**Pull request merged**